src/ideal/ideal.jl

```diff
@@ -840,6 +840,25 @@ function divrem(I::sideal{S}, G::sideal{S}; complete_reduction::Bool = false) wh
    return (smodule{S}(R,ptr_T), sideal{S}(R,ptr_Rest), smodule{S}(R,ptr_U))
 end
 
+@doc raw"""
+    divrem2(I::sideal{S}, G::sideal{S}; complete_reduction::Bool = false) where S <: SPolyUnion
+
+Computes a normal form of the generators of `I` by the generators of `G`,
+keeping track of the reduction. Returns a tuple (Quo, Rem, U) where
```

Comment:
The docstring for `divrem` does not include "keeping track of the reduction", is that an intentional difference, and if so, what does it signify?

Reply:
divrem uses internally lift and Rem may by different from reduce(I,G) (if G is not a Groebner basis),
while divrem2 uses internally reduce and Rem is reduce(I,G)